source/library/Signet/Unstable/Extra/EitherTest.hs

```diff
@@ -3,26 +3,25 @@ module Signet.Unstable.Extra.EitherTest where
 import qualified Control.Monad.Catch as Exception
 import qualified Data.Void as Void
 import qualified Signet.Unstable.Extra.Either as Either
-import qualified Signet.Unstable.Extra.Tasty as Tasty
-import Test.Tasty.HUnit ((@?=))
+import qualified Signet.Unstable.Type.Test as Test
 
-spec :: Tasty.Spec
-spec = Tasty.describe "Signet.Unstable.Extra.Either" $ do
-  Tasty.describe "hush" $ do
-    Tasty.it "works with Left" $ do
-      Either.hush (Left () :: Either () Void.Void) @?= Nothing
+spec :: (Monad tree) => Test.Test tree -> tree ()
+spec test = Test.describe test "Signet.Unstable.Extra.Either" $ do
```

Comment:
It would be nicer to enable `OverloadedRecordDot` and then say `test.describe`, but I was trying to avoid pulling in more extensions.